src/transformers/models/bloom/modeling_bloom.py

```diff
@@ -483,9 +477,7 @@ def __init__(self, config):
         self.bias_gelu = BloomBiasGelu()
 
     def forward(self, hidden_states, residual):
-        hidden_states = self.bias_gelu(
-            nn.functional.linear(hidden_states, self.dense_h_to_4h.weight), self.dense_h_to_4h.bias
```

Comment:
I think the whole purpose was that the other operation was jittable into a single fused kernel. that changes now no?

Reply:
I think that we finally do not want some operations to be jitable (after doing some benchmarks + from @stas00 's comment here : https://github.com/huggingface/transformers/pull/17474#issuecomment-1146169404
Also I had to define our own GeLU module to make the model torchscriptable (see comments on the class definition). Let me know what do you think!

Reply:
can you try with sequence length of 2048? and a bias dimension that's friendly to out usecase?